### ISSUE TITLE
Back Button

### ISSUE THREAD

**Issue by tserversbfs** [GAumala/MediaServer#1]
Please add add a back button in the playerTemplate.html file. 

I was going to post my code but it doesnt accept html inside this box. even with quote or code options.

**Comment by GAumala** [comment]
Hello! Thank you for your suggestion. If you want to post code in the comments you can use 3 back ticks like this:
~~~
``` html
<html><p>My html code</p></html>
``` 
~~~

However, I would appreciate if you submitted a pull request with the code for the back button, since I'm a little busy these days. 

If you want to contribute directly from your browser you can go to https://github.com/GAumala/MediaServer/blob/master/public/playerTemplate.html and click the button that looks like a pencil

![screenshot from 2018-11-25 20-10-38](https://user-images.githubusercontent.com/5729175/48987574-5c867b00-f0ee-11e8-8034-de6e2dc9d9ed.png)

This will let you edit `playerTemplate.html` and submit a PR.

cheers!

**Comment by tserversbfs** [comment]
Ok. I think I did that correctly. This is my first pull request. Please let me know if it should have been done another way, or a better way.

Also, I have been thinking about adding a base URL. Right now I have to access the MediaServer via its own subdomain. I much rather be able to access it via www.example.com/some/other/url

Thank you for your time and your work.

**Comment by GAumala** [comment]
Yeah, there's really no reason to not be able to do that. It's only a matter of going through the templates and replacing all absolute paths with relative paths.

For example, `playerTemplate.html` references a stylesheet like this.
``` html
 <link href="/video-js.css" rel="stylesheet">
```

changing the link to `../video-js.css` should do the trick.

**Comment by tserversbfs** [comment]
That is what I thought. lol. But it didn't work but had other things that were more important. I will try again over the next few days.

**Comment by tserversbfs** [comment]
Update on adding a base URL. I tried your suggestion and it worked except for the most important part. :P In the list of videos the URL for the movie was still "/watch?v=". 

so, In streamListTemplate.html I hardcoded my base URL as a test. href="../watch?v={{.Key}}" became href="../base/URL/watch?v={{.Key}}". partial success. All videos listed has the correct URL! 

When I clicked on the video link to play the movie the URL reverted to: "/watch?v=". The URL in streamListTemplate.html must be just for show? 

I will look again when I get a moment. 

Thanks!
